scripts/run_tests.sh

```diff
@@ -36,4 +36,7 @@ npm run --silent frontend-build:production
 display_result $? 1 "Build of front end static assets"
 
 nosetests -v -s --with-doctest
-display_result $? 3 "Unit tests"
+display_result $? 3 "Python unit tests"
+
+npm test
+display_result $? 3 "JavaScript unit tests"
```

Comment:
If we're using different exit statuses we should probably use a different one for the JavaScript tests as well so that we can tell the difference between Python and JavaScript test failures.